x-pack/plugins/rule_registry/common/parse_technical_fields.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { parseTechnicalFields } from './parse_technical_fields';
+
+describe('parseTechnicalFields', () => {
+  it('parses an alert with default fields without error', () => {
+    const ALERT_WITH_DEFAULT_FIELDS = {
+      '@timestamp': ['2021-12-06T12:30:59.411Z'],
+      'kibana.alert.status': ['active'],
+      'kibana.alert.duration.us': ['488935266000'],
+      'kibana.alert.reason': ['host.uptime has reported no data over the past 1m for *'],
+      'kibana.alert.workflow_status': ['open'],
+      'kibana.alert.rule.uuid': ['c8ef4420-4604-11ec-b08c-c590e7b8c4cd'],
+      'kibana.alert.rule.producer': ['infrastructure'],
+      'kibana.alert.rule.consumer': ['alerts'],
+      'kibana.alert.rule.category': ['Metric threshold'],
+      'kibana.alert.start': ['2021-11-30T20:42:04.145Z'],
+      'kibana.alert.rule.rule_type_id': ['metrics.alert.threshold'],
+      'event.action': ['active'],
+      'kibana.alert.instance.id': ['*'],
```

Comment:
kibana.alert.instance.id is an experimental field and they get parsed by parseExperimentalFields instead. Here's the PR that introduced the experimental fields https://github.com/elastic/kibana/pull/120904. Let me know if you have any questions regarding this